config/jobs/kubernetes-sigs/cloud-provider-azure/cloud-provider-azure-config.yaml

```
@@ -706,3 +706,55 @@ periodics:
       value: /kubemark-performance-tests-on-azure
     - name: ARTIFACTS
       value: $(WORKSPACE)/_artifacts
+
+# conformance test against kubernetes master branch with `azure` ipv6
+- interval: 6h
+  name: ci-kubernetes-azure-conformance-ipv6
+  decorate: true
+  labels:
+    preset-service-account: "true"
+    preset-azure-cred: "true"
+    preset-dind-enabled: "true"
+  extra_refs:
+  - org: kubernetes
+    repo: kubernetes
+    base_ref: master
+    path_alias: k8s.io/kubernetes
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20200305-26ebc13-master
+      command:
+      - runner.sh
+      - kubetest
+      args:
+      - --test
+      - --up
+      - --down
+      - --build=quick
+      - --dump=$(ARTIFACTS)
+      # Azure-specific test args
+      - --deployment=aksengine
+      - --provider=skeleton
+      - --aksengine-agentpoolcount=2
+      - --aksengine-admin-username=azureuser
+      - --aksengine-creds=$AZURE_CREDENTIALS
+      - --aksengine-orchestratorRelease=1.18
+      - --aksengine-mastervmsize=Standard_DS2_v2
+      - --aksengine-agentvmsize=Standard_D4s_v3
+      - --aksengine-deploy-custom-k8s=True
+      - --aksengine-location=eastus2
+      - --aksengine-public-key=$AZURE_SSH_PUBLIC_KEY_FILE
```

`$AZURE_SSH_PUBLIC_KEY_FILE` would be parsed as plain text instead of environment variables. Try using:

```suggestion
      - --aksengine-public-key=$(AZURE_SSH_PUBLIC_KEY_FILE)
```